src/main/webapp/app/entities/event-attendee/event-attendee.scss

```diff
@@ -0,0 +1,4 @@
+.icon-btn {
+  padding: 0px !important;
+  font-size: 14px;
+}
```

Comment:
Can move this to event-module.scss
Can have a enhancement issue for it

Reply:
ok, will change it in new issue